ui/src/TreeTablePage.vue

```diff
@@ -43,6 +44,7 @@ const tableInstance = ref<PlAgDataTableController>();
   <PlBlockPage>
     <template #title>Trees Table</template>
     <template #append>
+      <PlAgDataTableToolsPanel />
       <PlBtnGhost @click.stop="() => tableInstance?.exportCsv()">
```

Comment:
`exportCsv` should be also part of the panel?

Reply:
It will be done in the near future